src/tracking-buffer/writable-tracking-buffer.js

```diff
@@ -273,7 +273,9 @@ module.exports = class WritableTrackingBuffer {
   }
 
   writeMoney(value: number) {
-    this.writeInt32LE(Math.floor(value * SHIFT_RIGHT_32));
+    //floor(-1.9) === -2
+    //-1.9|0 === -1
+    this.writeInt32LE((value * SHIFT_RIGHT_32) | 0);
```

Comment:
This would be simpler. I think `this.writeInt32LE` is already doing the `| 0` operation internally. 👍 

```suggestion
    this.writeInt32LE(value * SHIFT_RIGHT_32);
```